jsonnet/components/prometheus-adapter-audit.libsonnet

```diff
@@ -0,0 +1,117 @@
+// NOTE: audit rules are copied from openshift/library-go/pkg/operator/apiserver/audit/manifests/*.yaml
+// to maintain the same UX as OpenShift apiserver
```

Comment:
We shouldn't copy the manifests directly since we are exposing audit logs on different resources than them. In our case, we will only expose audit logs for NodeMetrics and PodMetrics, so it doesn't really make sense to have a policy about routes and oauth since the adapter will never be responsible for these resources.